ustriage/task.py

```diff
@@ -107,13 +113,78 @@ class Task:
     NOWORK_BUG_STATUSES = []
     OPEN_BUG_STATUSES = []
 
-    def __init__(self):
+    def __init__(self, lp_task=None):
```

Comment:
variable name nitpick - it's a bit hard to tell that lp_task refers to a url when initializing a Task object externally. Changing the name to `lp_task_url` or something similar may help with that.

Reply:
Actually, it isn't a url, it's a Launchpad API 'task' object.  It gets converted to a url via str(task):

```
            task_elements = str(lp_task).split('/')
            self.distro = task_elements[4]
            self.source_package_name = task_elements[-3]
            self.series = task_elements[5]
            if self.series == '+source':
                self.series = '-devel'
            self.obj = lp_task
```
So self.obj can't point to a URL, it has to point to the LP object.

Sorry, I'll prioritize improving the class documentation in the next branch.  

I probably could have left this out of the branch, it actually is required by the next step of refactoring which I pushed to the next branch, which is to cleanup the duplicate logic I added in 443305f5.  I found that doing this broke code several places so the next step needs a bit more TLC.  So nothing actually uses this constructor logic yet.